radar/languages_frameworks/kotlin.yml

```diff
@@ -0,0 +1,19 @@
+version: 1
+name: Kotlin
+blip:
+  - version: "2.6"
+    ring: ADOPT
```

Comment:
Is this used in production in any product?

Reply:
It is not in production yet, but our new Kafka streams applications are built using Kotlin and Micronaut. Several here also have previous experience running Kotlin in production, and have good experiences with it.